client/src/pages/HomePage/useStyles.ts

```diff
@@ -0,0 +1,41 @@
+import { makeStyles } from '@mui/styles';
+import { Theme } from '@mui/material/styles';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    marginTop: 50,
+    minHeight: '100vh',
+    '& .MuiInput-underline:before': {
+      borderBottom: '1.2px solid rgba(0, 0, 0, 0.2)',
+    },
+  },
+  flexContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifiyContent: 'center',
+    height: '90vh',
+  },
+  inputBox: {
+    width: '50%',
+  },
+  landingPhotoBox: {
+    width: '50%',
+    backgroundColor: 'purple',
```

Comment:
We wouldn't want to change colors outside of MUI's theme, unless we absolutely need to